### PR TITLE
Disable building tests when including as subproject

### PR DESCRIPTION
**Pull request by mohe2015**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -20,8 +20,10 @@ add_library(${PROJECT_NAME} ${SRC})
 target_include_directories(${PROJECT_NAME} INTERFACE src)
 
 # test
-enable_testing()
-add_subdirectory(test)
-find_package(Sanitizers)
-add_sanitizers(${PROJECT_NAME} test.exe)
-add_sanitizers(${PROJECT_NAME} test_miniz.exe)
+if (NOT KUBA_ZIP_DISABLE_TESTS)
```
Comment (kuba--) [inline CMakeLists.txt]: Maybe `CMAKE_DISABLE_TESTING`

Comment (mohe2015) [reply CMakeLists.txt]: But that may clash with names of other projects...

Comment (kuba--) [reply CMakeLists.txt]: But if someone set `CMAKE_DISABLE_TESTING`, I suppose the intention was to disable all ctests, so also this one.

Comment (mohe2015) [reply CMakeLists.txt]: Okay, I changed it. You can squash merge now if you want to.

**Pull request merged**